parser/parser.go

```diff
@@ -28,7 +28,7 @@ func Parse(reader io.Reader) ([]Command, error) {
 		line := scanner.Bytes()
 
 		fields := bytes.SplitN(line, []byte(" "), 2)
-		if len(fields) == 0 {
+		if len(fields) == 0 || len(fields[0]) == 0 {
```

Comment:
This was a bug from earlier where an empty line will be parsed into a [][]byte of length 1 where fields[0] is the empty line